features/accesskey.yml

```
@@ -0,0 +1,9 @@
+name: AccessKey
+description: The `accessKey` global attribute and property gives a hint for generating a keyboard shortcut for the current element. The attribute value must consist of a single printable character.
```

I feel like these global attribute feature descriptions should only mention "global attribute", and not "property", to keep things simple. But let me know if you think it's important to keep both.

```suggestion
description: The `accessKey` global attribute gives a hint for generating a keyboard shortcut for the current element. The attribute value must consist of a single printable character.
```

---

I called out above that while it's more correct but I didn't like it and wanted feedback, so am totes fine w/ this, thanks!